python/apps/taiga/src/taiga/stories/stories/api/__init__.py

```diff
@@ -235,6 +235,6 @@ async def delete_story(
 async def get_story_or_404(project_id: UUID, ref: int) -> Story:
     story = await stories_services.get_story(project_id=project_id, ref=ref)
     if story is None:
-        raise ex.NotFoundError(f"Story {ref} does not exist in project {project_id}")
+        raise ex.NotFoundError("Story does not exist")
```

Comment:
Why do you change that?

Reply:
As we can't return references for projects and workspaces I was initially tempted to act this way in other endpoints, as it's not really required and it would be more consistant. But then I realized I should change many endpoints, and it wasn't the aim of this US. So I left just this file changed by error.
I'll left it returning the previous references.

Reply:
I can't return the project b64ID, but I can return the story ref. I'll left it this way